modules/transaction-payment/src/lib.rs

```diff
@@ -639,22 +639,6 @@ pub mod module {
 			call.dispatch(origin)
 		}
 
-		/// Wrap call with fee paid by other account
-		#[pallet::call_index(5)]
```

Comment:
@xlc should we leave func name and call_index as comment here?

Reply:
good point. that will ensure we never reuse the index